app/dao/notifications_dao.py

```diff
@@ -20,6 +20,7 @@
     EMAIL_TYPE,
     KEY_TYPE_TEST,
     NOTIFICATION_CREATED,
+    NOTIFICATION_DELIVERED,
```

Comment:
This will not work, as these strings simply do not exist any more. This needs to be adjusted to use app.enums and load/use the correct enums.